src/process_utility.c

```diff
@@ -1011,6 +1011,15 @@ process_truncate(ProcessUtilityArgs *args)
 		}
 		else
 		{
+			Chunk *chunk = ts_chunk_get_by_relid(relid, false);
+			if (chunk != NULL)
+			{
+				ereport(ERROR,
+						(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+						 errmsg("Truncating chunks not supported"),
+						 errhint("Please drop the corresponding chunk instead.")));
```

Comment:
No need to be courteous :)

```suggestion
						 errhint("Drop the corresponding chunk instead.")));
```